dts/bindings/usb/usb-audio-hp.yaml

```diff
@@ -10,6 +10,10 @@ compatible: "usb-audio-hp"
 include: usb-audio.yaml
 
 properties:
+  frequency:
+    type: array
+    default: [48000]
```

Comment:
I think these should all be `required` and no default should be given, no?

I'm referring to this property, as well as the `mic-frequency` and `hp-frequency` properties in usb-audio-hs.yaml, and the `frequency` property in usb-audio-mic.yaml.

It seems like these are all generic hardware bindings that shouldn't assume this frequency is supported by default, but I'm not familiar with this area and may be missing something.